### ISSUE TITLE
Pass raw event values to front-end css

### ISSUE THREAD

**Issue by bcipolli** [codeforamerica/citygram#244]
Currently, the event information is combined somewhere else (server-side?) and sent to the front-end 
![image](https://cloud.githubusercontent.com/assets/4072455/12047441/4c8513e6-ae80-11e5-8ff0-cd31ad93c21b.png)

**Comment by bigfleet** [comment]
Here's the relevant JS code: https://github.com/codeforamerica/citygram/blob/master/app/assets/js/scripts.js#L317

**Comment by bcipolli** [comment]
Yep. Sadly, the HTML generation code must be found within the middleware that pulls data into the database; by the time it gets to Ruby or the JS front-end, it's a HTML blob already. This makes it quite tenuous to reformat in the front-end (hence the retracted PR).

**Comment by bigfleet** [comment]
I'm going to point to this "request for comment" type page:, since this is something I'm certainly open to.  https://github.com/codeforamerica/citygram/wiki/GeoJSON-Enhancements

**Comment by dsummersl** [comment]
Related to this, I've opened a PR to make the sandiego feed text: https://github.com/opensandiego/open-dsd-notifications/pull/15 --- does this fix that?

**Comment by dsummersl** [comment]
Following up on the RCP type page WRT https://github.com/opensandiego/open-dsd-notifications/pull/15 and https://github.com/codeforamerica/citygram/issues/247:

I think to fix issues around links/text the publisher format needs to separate out the concepts publishers are using now:
 * title: a text summary of the event without street, link, or date information.
 * href: a link to the original event.
 * address: a human readable rough address.
 * created: timestamp when the event was created in some version of ISO8601

Perhaps using some derivation of the [permitdata.org](https://github.com/open-data-standards/permitdata.org/wiki/Core-Permits-Dataset-Requirements) the [open311](http://wiki.open311.org/GeoReport_v2/) specs would be helpful. Looks like they both have concepts like...
 * description: a fuller description of the event.
 * durations: what time period the event covers start to finish.
 * contact: an email to reach out to regarding the particular event.
